cloudbuild.yaml

```diff
@@ -10,8 +10,21 @@ steps:
   # Build Docker image based on current Dockerfile, if necessary.
   - name: 'gcr.io/cloud-builders/docker'
     id: build_image
+    waitFor: ['pull_image']
     timeout: 10m
     args: ['build', '--pull', '--cache-from=gcr.io/oak-ci/oak:latest', '--tag=gcr.io/oak-ci/oak:latest', '.']
+  # Init .git repository used by check_generated
+  # Workaround for https://github.com/GoogleCloudPlatform/cloud-builders/issues/236
+  - name: 'gcr.io/cloud-builders/git'
+    id: add_git_directory
+    entrypoint: 'bash'
+    waitFor: ['pull_image']
+    timeout: 5m
+    args:
```

Comment:
Maybe it would be more consistent to check in a separate script `git_init` and rename `check_generated` to `git_check_diff` or similar. WDYT?

Reply:
Done